flang/tools/flang-driver/CMakeLists.txt

```diff
@@ -19,8 +19,6 @@ add_flang_tool(flang-new
   # These libraries are used in the linker invocation generated by the driver
   # (i.e. when constructing the linker job). Without them the driver would be
   # unable to generate executables.
```

Comment:
You can probably also remove the comment

Reply:
Good point.  Will do.

Reply:
Done